snapcraft_legacy/internal/repo/_deb.py

```diff
@@ -476,6 +476,7 @@ def fetch_stage_packages(
         base: str,
         stage_packages_path: pathlib.Path,
         target_arch: str,
+        filters: Set[str] = set(),
```

Comment:
```suggestion
        package_filters: Set[str] = set(),
```

Reply:
Done in 89152e8870f3218e54f5d9ea8d2152cedf867e8d .